Scripts/CharacterBody2D.gd

```diff
@@ -409,15 +406,20 @@ func dashActivation():
 
 # Power Ups Uses
 func useDash(delta):
+	if (get_local_mouse_position() - to_local(position)).x > 0 and dashDirection == 0:
+		dashDirection = 1
+	elif  dashDirection == 0:
+		dashDirection = -1
 	
 	if dashTime > 0:
 		dashTime -= delta
-		target_velocity = get_local_mouse_position().normalized() * 1000
+		target_velocity = Vector2(300*dashDirection, 0)
```

Comment:
Do you want to make this a function of time delta so that the game cycle stays in sync when dashing? Otherwise i might see the character clipping through a collider